tasks/generate/cache.rake

```diff
@@ -0,0 +1,28 @@
+namespace :generate do
+  desc "Generate a cache of lexers"
+  task :cache do
+    require "json"
+    require "rouge/util"
+    require "rouge/token"
+    require "rouge/lexer"
+    require "rouge/regex_lexer"
+    require "rouge/template_lexer"
+
+    paths = FileList["lib/rouge/lexers/*.rb"]
+    
+    paths.each { |path| Rouge::Lexers.load_lexer(File.basename(path)) }
+
+    proxies = Array.new
+
+    Rouge::Lexers.instance_variable_get(:@_loaded_files).each do |lexer,path|
+      proxies << { "path"      => path,
+                   "name"      => lexer.name.split("::").last,
+                   "tag"       => lexer.instance_variable_get(:@tag),
```

Comment:
Is there a reason `lexer.tag` is not sufficient here...?

Reply:
Nope! I think this may have been me using `@title` at an earlier point in the evolution of this code. Thanks for the pick up :)